include/git2/transaction.h

```diff
@@ -8,6 +8,14 @@
 #define INCLUDE_git_transaction_h__
 
 #include "common.h"
+
+/**
+ * @file git2/transaction.h
+ * @brief Git transaction routines.
+ * @defgroup Git transaction routines.
```

Comment:
We don't actually use these blocks for anything with our current documentation generator, so it should just go away.